core/iwasm/aot/aot_runtime.c

```diff
@@ -1413,6 +1413,8 @@ aot_call_function(WASMExecEnv *exec_env, AOTFunctionInstance *function,
                   unsigned argc, uint32 argv[])
 {
     AOTModuleInstance *module_inst = (AOTModuleInstance *)exec_env->module_inst;
+    module_inst->e->common.exec_env = exec_env;
```

Comment:
Should `((AOTModuleInstanceExtra *)module_inst->e)->common.exec_env`, note that AOTModuleInstanceExtra is different from WASMModuleInstanceExtra

Reply:
And could you put the line after L1446 `wasm_exec_env_set_thread_info(exec_env)` and add comment?